### DISCUSSION TITLE
Can I please get recs and comments for 1920x1080 screen resolution in 86box guest?

### DISCUSSION THREAD

**Discussion by HanHeld** [86Box/86Box#2084] [Q&A] [answered]
Hi, I'd like to know what video cards to pick in 86box; preferably ones that can work with 486 or 386 models which can do 1920x1080 screen resolution or as close as possible -and also if there are windows 3.0 or 3.1 drivers that are able to do that (and for which cards). 

Of course the "resizable window" option allows you to kinda fake it, but if there's a way to run windows 3.x at that resolution it would be neat to be able to.

Thanks for the emulator (of course!) and for the advice

**Comment by Cacodemon345** [comment, marked as answer]
I don't think graphics cards of that era were able to output at 1920x1080 if at all.

**Comment by HanHeld** [comment]
I was afraid that might be the case (not surprising since the average for 1990 seems to top out at 1 or 2 meg video cards). I know I can get reasonable high resolution with the s3 cards and windows 3.1 -I assume 3.0 is a lost cause. So I'll stick with either S3 or Cirrus.

Thanks!

    On Wednesday, February 9, 2022, 09:40:06 AM AKST, Cacodemon345 ***@***.***> wrote:  
 
 


I don't think graphics cards of that era were able to output at 1920x1080 if at all.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by leuamS** [comment]
I used the Voodoo 3 in Windows XP and that pushed 1920x1080

**Reply by leuamS** [reply]
oh wait this is talking about Windows 3.X
